modules/migration-changing-migration-plan-limits.adoc

```diff
@@ -4,38 +4,28 @@
 // * migration/migrating_4_1_4/migrating-applications-with-cam-4-1-4.adoc
 // * migration/migrating_4_2_4/migrating-applications-with-cam-4-2-4.adoc
 [id='migration-changing-migration-plan-limits_{context}']
-= Changing migration plan limits for large migrations
+= Increasing migration plan limits for large migrations
 
-You can change the migration plan limits for large migrations.
+You can increase the migration plan limits for namespaces, Pods, and persistent volumes (PVs) for large migrations.
 
 [IMPORTANT]
 ====
-Changes should first be tested in your environment to avoid a failed migration.
+You must test these changes before you perform a migration in a production environment.
 ====
 
 A single migration plan has the following default limits:
 
 * 10 namespaces
-+
-If this limit is exceeded, the CAM web console displays a *Namespace limit exceeded* error and you cannot create a migration plan.
-
 * 100 Pods
+* 100 PVs
 +
-If the Pod limit is exceeded, the CAM web console displays a warning message similar to the following example: *Plan has been validated with warning condition(s). See warning message. Pod limit: 100 exceeded, found: 104*.
-
-* 100 persistent volumes
-+
-If the persistent volume limit is exceeded, the CAM web console displays a similar warning message.
+If these limits are exceeded, the migration plan displays warning messages when you save it.
```

Comment:
Is the `+` sign needed here to indent/attach the explanation for exceeding limits? Since the sub-descriptions for each limit have been removed, the last sentence appears to be describing all three limits now. The way it renders now, it makes it seem like this sentence only applies for the 100 PVs.